pkg/internal/checkup/executor/executor.go

```diff
@@ -116,6 +121,28 @@ func (e Executor) runTestpmd(vmiName string) error {
 	return nil
 }
 
+func (e Executor) clearStatsTestpmd(vmiName string) error {
+	const batchTimeout = 30 * time.Second
+
+	const testpmdPromt = "testpmd> "
```

Comment:
We already have this const in `runTestpmd()`.
I suggest to increase its scope and use it in both methods.

Reply:
I plan to do it a a future PR.

#2DO RAM